### PR TITLE
K8s

### PR DESCRIPTION
**Pull request by cmyui**

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+types-aiobotocore[s3]
```
Comment (cmyui) [inline requirements-dev.txt:1]: @tsunyoku this is the way

File: app/api/screenshots.py [hunk 2]
```diff
@@ -14,40 +14,36 @@
...
 def gen_rand_str(len: int) -> str:
-
-    return "".join(random.choice(AV_CHARS) for _ in range(len))
+    return "".join(random.choice(FILENAME_CHARSET) for _ in range(len))
 
 
 async def upload_screenshot(
     user: User = Depends(authenticate_user(Form, "u", "p")),
-    screenshot_file: UploadFile = File(None, alias="ss"),
+    screenshot_file: UploadFile = File(..., alias="ss"),
```
Comment (cmyui) [inline app/api/screenshots.py:46]: bug

File: Dockerfile [hunk 3]
```diff
@@ -1,6 +1,17 @@
...
 
+RUN apt update && \
+    apt install -y default-mysql-client
+
+COPY . /srv/root
+WORKDIR /srv/root
+
+EXPOSE 80
+
 CMD ["/srv/root/main.py"]
```
Comment (cmyui) [inline Dockerfile:17]: not sure wtf i was doing when i originally made this dockerfile

**Comment by cmyui** [comment]
as a separate step, we'll need to ensure all replays are migrated from ftp server to s3. then we can delete ftp server.

**Comment by cmyui** [comment]
hmm @tsunyoku we should probably try merging this, even if we're not going for k8s yet

good to start moving our stuff to s3 anyways

**Comment by cmyui** [comment]
we should also make a habit of building things to NOT use disk going forward, to make future efforts simpler

**Comment by tsunyoku** [comment]
@cmyui yes and yes

**Pull request closed without merging**